string chesspiece
string field
---
bool success
